frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -1,9 +1,11 @@
 name: {{FRAMEWORK_NAME}}
 scheduler:
   principal: {{SERVICE_PRINCIPAL}}
+  user: {{TASKCFG_ALL_TASK_USER}}
```

Comment:
Oh... I didn't notice that. Before you do that, @nickbp @gabrielhartmann is that required to make strict work?